modules/google-maps/src/google-maps-overlay.ts

```diff
@@ -247,14 +247,6 @@ export default class GoogleMapsOverlay {
       // a second repaint
       deck.needsRedraw({clearRedrawFlags: true});
 
-      // Workaround for bug in Google maps where viewport state is wrong
-      // TODO remove once fixed
-      setGLParameters(gl, {
-        viewport: [0, 0, gl.canvas.width, gl.canvas.height],
-        scissor: [0, 0, gl.canvas.width, gl.canvas.height],
-        stencilFunc: [gl.ALWAYS, 0, 255, gl.ALWAYS, 0, 255]
-      });
-
```

Comment:
these are managed by beginRenderPass now i recon ... and i think they are not needed now

Reply:
Resizing the window results in the overlay being the wrong aspect. Could you check if this is due to removing this code?
<img width="459" alt="Screenshot 2023-12-13 at 13 32 58" src="https://github.com/visgl/deck.gl/assets/453755/3eb12b3b-1208-4758-8177-9968353d018c">